lib/coinbase.rb

```diff
@@ -27,41 +27,47 @@ def self.configuration
   end
 
   # Configures the Coinbase SDK.
+  # @return [String] A string indicating successful configuration
   def self.configure
     yield(configuration)
 
     raise InvalidConfiguration, 'API key private key is not set' unless configuration.api_key_private_key
     raise InvalidConfiguration, 'API key name is not set' unless configuration.api_key_name
+
+    'Successfully configured Coinbase SDK'
```

Comment:
Do we want to print this out when `debug_api` is not set?

I think it's a bit weird to return a string here fwiw

Reply:
Yes lets remove

Reply:
Let's do a full pass in the next release and clean these up!